src/js/tabs/pid_tuning.js

```diff
@@ -492,6 +492,10 @@ pid_tuning.initialize = function (callback) {
             $('.rpmFilter').hide();
         }
 
+        if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```

Comment:
It should be 1.45??

Reply:
Firmware PR is tagged as 4.3.2 not 4.4

Reply:
Ok so, we need to fix it first, it is taged after merge and is merged on ly in master

Reply:
Fixed.